task-06/README.md

```diff
@@ -26,29 +26,32 @@ USEMODULE += gnrc_sock_udp
    udp                  send udp packets
    udps                 start udp server
    reboot               Reboot the node
+   version              Prints current RIOT_VERSION
+   pm                   interact with layered PM subsystem
    ps                   Prints information about running threads.
    ping6                Ping via ICMPv6
+   ping                 Ping via ICMPv6
```

Comment:
hmm .. I am seeing this just now. Having two times the same help message looks strange to me. Something alone the lines of `see ping6` or `behaves like ping6` would be IMO better than duplicating the help text. It's not part of this PR, though.

Reply:
You are welcome to change that. I'm not sure however that is the correct help text for that alias. `Alias for ping6` maybe?